src/main/java/com/minelittlepony/hdskins/skins/GameSession.java

```diff
@@ -0,0 +1,40 @@
+package com.minelittlepony.hdskins.skins;
```

Comment:
Why do we need our own session stuff?

Reply:
Minecraft's `Session` class is client-side only.

Reply:
What do we need the session on the server for anyway?

Reply:
Because it's an argument to the upload method. If we don't include it, the game will try and fail to load it.